map.json

```diff
@@ -37,7 +37,7 @@
                 {
                  "name":"exitSceneUrl",
```

Comment:
This should actually be:

```suggestion
                 "name":"exitUrl",
```

exitSceneUrl is only supposed to work on JSON map URLs